testsuite/classlibrary/TestNodeProxy.sc

```diff
@@ -157,5 +157,49 @@ TestNodeProxy : UnitTest {
 		server.remove;
 	}
 
+	test_control_fade {
+		var server = Server(this.class.name);
+		var result, proxy, timeout;
+		var cond = Condition.new;
+		var fadeTime = 0.1;
+		var waitTime = (fadeTime + (server.latency * 2) + 1e-2);
+
+		server.bootSync;
+
+		proxy = NodeProxy(server);
+		proxy.source = { DC.kr(2000) };
+		proxy.fadeTime = fadeTime;
+
+		proxy.source = 440;
+		waitTime.wait;
+
+		OSCFunc({ cond.unhang }, '/c_set');
```

Comment:
this needs to be assigned to a variable and `free`d after `cond.hang`. everything else here looks good!

Reply:
@brianlheim should be fixed now. Thanks!